translations/resources.tex/pl.po

```diff
@@ -76,18 +93,23 @@ msgid ""
 "Scenario’s setup.  Resources can also be \\hyperlink{Trading}{traded}.  "
 "There's no limit to the amount of Resources you can have."
 msgstr ""
+"Gracze rozpoczynają każdy Scenariusz z ilością Zasobów określoną w Przygotowaniu Scenariusza.  "
+"Zasobami można również \\hyperlink{Trading}"
+"{handlować}. Nie ma ograniczeń w ilości posiadanych Zasobów."
 
 #. type: textcolor{#2}
 #: sections/resources.tex:37
 msgid "\\textbf{\\textit{Resource Production Tracker}}"
-msgstr ""
+msgstr "\\textbf{\\textit{Tor przychodu}}"
 
 #. type: Plain text
 #: sections/resources.tex:49
 msgid ""
 "Possible Resource Die\\index{Resource Die}\\index{Dice} "
 "\\includesvg[height=10px]{\\svgs/resource_die.svg} results:"
 msgstr ""
+"Rzucając \\index{Kość Zasobów}\\index{Kość} Kością Zasobów "
```

Comment:
Nie mam pewności czy drugi index też nie powinien zawierać Kość Zasobów, ale w oryginalnym tekście jest to też rozróżnione